go/README.md

```diff
@@ -14,22 +14,42 @@
 
 ```
 Usage of ./networkqualityd:
-  -config-port int
-    	The base port to listen on (default 4043)
+  -announce
+        announce this server using DNS-SD
   -cert-file string
-    	cert to use
-  -debug
-    	enable debug mode
+        cert to use
   -config-name string
-    	domain to generate config for (default "networkquality.example.com")
+        domain to generate config for (default "networkquality.example.com")
+  -context-path string
+        context-path if behind a reverse-proxy
+  -debug
+        enable debug mode
+  -enable-cors
+        enable CORS headers
+  -enable-h2c
+        enable h2c (non-TLS http/2 prior knowledge) mode
+  -enable-http2
+        enable HTTP/2 (default true)
+  -enable-http3
+        enable HTTP/3
+  -enable-insecure-listener
+        enable insecure listener
+  -insecure-public-port int
+        The port to listen on for measurement accesses (default 4080)
   -key-file string
-    	key to use
+        key to use
   -listen-addr string
-    	address to bind to (default "localhost")
+        address to bind to (default "localhost")
   -public-name string
-    	host to generate config for
+        host to generate config for (same as -config-name if not specified)
+  -public-port int
```

Comment:
I am absolutely *not* worried about it, but I was just curious what made you decide to change?

Reply:
which part? setting both `config-name` and `public-name` is a minor pain to think about when not running behind a proxy.

and the public-port was renamed to align with the addition of a second port (`insecure-public-port`)